### PR TITLE
Update obsolete/broken SailPoint docs links

### PR DESCRIPTION
**Pull request by kimsauce**
## Purpose of this pull request

Update obsolete/broken SailPoint docs links 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions, .clabot
- [x] Update Content - Revisions, updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

### CODE REVIEW

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/sailpoint-source.md [hunk 1]
```diff
@@ -80,7 +80,7 @@ To configure a Duo Source:
 
 ## JSON schema
 
-Sources can be configured using UTF-8 encoded JSON files with the Collector Management API. See [how to use JSON to configure Sources](/docs/send-data/use-json-configure-sources) for details. 
+Sources can be configured using UTF-8 encoded JSON files with the Collector Management API. See [How to use JSON to configure Sources](/docs/send-data/use-json-configure-sources) for details.
```
Comment (jpipkin1) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/sailpoint-source.md:83]:
```suggestion
Sources can be configured using UTF-8 encoded JSON files with the Collector Management API. See [Use JSON to Configure Sources](/docs/send-data/use-json-configure-sources) for details. 
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/sailpoint-source.md [hunk 2]
```diff
@@ -95,13 +95,13 @@ Sources can be configured using UTF-8 encoded JSON files with the Collector Ma
...
+| fields | JSON Object | No | `null` | JSON map of key-value fields (metadata) to apply to the Collector or Source. Use the boolean field `_siemForward` to enable forwarding to SIEM.|`{"_siemForward": false, "fieldA": "valueA"}` |
 | org_name | String | Yes | `null` | Provide your SailPoint customer-specific organization name, such as `{organization}.identitynow.com`.	| |
 | client_id | String | Yes | `null` | Provide the SailPoint client ID you want to use to authenticate collection requests.	| |
 | client_secret | String | Yes | `null` | Provide the SailPoint secret you want to use to authenticate collection requests.|	 |
 | supported_apis | Array of strings | Yes | `null` | Define one or more of the available APIs to collect: Events, and Users.<br/>For example, for both you'd use: `["Events","Users"]` |  |
 
-See how to [create processing rules using JSON](/docs/send-data/use-json-configure-sources).
+See [How to create processing rules using JSON](/docs/send-data/use-json-configure-sources).
```
Comment (jpipkin1) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/sailpoint-source.md:104]:
```suggestion
See [Create processing rules using JSON](/docs/send-data/use-json-configure-sources/#creating-processing-rules-using-json).
```

**Review by jpipkin1 — APPROVED** [review]
Minor things.

**Comment by kimsauce** [comment]
> Minor things.

thank you, @jpipkin1 !

**Pull request merged**